app/deliver_grant_funding/templates/deliver_grant_funding/grant_team/grant_user_list.html

```diff
@@ -42,7 +42,7 @@ <h1 class="govuk-heading-l">
 
   {% set table_pending_signin_rows = namespace(items=[]) %}
   {% if users %}
-    {% for user in users if user.name|lower == '' %}
+    {% for user in users if not user.name %}
```

Comment:
This feels fairly similar to working out grant users in the view, so can we make logged in/pending users a property on the grant please as per here https://github.com/communitiesuk/funding-service/pull/276/files#diff-43a053fff309c701842571c7d0eb13a1b24337b2055e5c7437276f7a6be0e008R194